ironfish/src/blockchain/nullifierSet/nullifierSet.test.ts

```diff
@@ -0,0 +1,81 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import {
+  createNodeTest,
+  useAccountFixture,
+  useBlockWithTx,
+  useMinerBlockFixture,
+} from '../../testUtilities'
+import { NullifierSet } from './nullifierSet'
+
+describe('NullifierSet', () => {
+  const nodeTest = createNodeTest()
+
+  it('connects blocks and checks that nullifiers are included', async () => {
+    const { node, chain } = nodeTest
+
+    const set = new NullifierSet({ db: chain.db, storeName: 'nu-test' })
+
+    await chain.open()
+
+    const accountA = await useAccountFixture(node.wallet, 'accountA')
+
+    const block2 = await useMinerBlockFixture(node.chain, 2, accountA)
+    await expect(node.chain).toAddBlock(block2)
+
+    await node.wallet.updateHead()
+
+    const { block: block3 } = await useBlockWithTx(node, accountA, accountA, false)
+    await expect(node.chain).toAddBlock(block3)
+
+    await node.wallet.updateHead()
+
+    const { block: block4 } = await useBlockWithTx(node, accountA, accountA, false)
+    await expect(node.chain).toAddBlock(block4)
+
```

Comment:
Maybe `await node.wallet.updateHead()`? I guess it doesn't do anything anyway

Reply:
added